divi/src/BlockDiskAccessor.cpp

```diff
@@ -7,6 +7,7 @@
 #include <chainparams.h>
 #include <txmempool.h>
 #include <txdb.h>
+#include <main.h>
```

Comment:
Can we remove this include and simplify it with a smaller set of includes or externs?

Reply:
done